lib/ace/edit_session/bracket_match.js

```diff
@@ -37,10 +37,77 @@ var Range = require("../range").Range;
 
 function BracketMatch() {
 
+
+    this.getBracketsCoor = function(position) {
```

Comment:
I suggest renaming this to `findSurroundingBrackets` -- this sounds consistent with the existing functions